map/create-and-save-map/src/main/java/com/esri/samples/create_and_save_map/CreateAndSaveMapController.java

```diff
@@ -147,12 +150,11 @@ void authenticate() {
           } else if (portal.getLoadStatus() == LoadStatus.FAILED_TO_LOAD) {
 
             // show alert message on error
-            showMessage("Authentication failed", portal.getLoadError().getMessage(), Alert.AlertType.ERROR);
+            new Alert(Alert.AlertType.ERROR, "Authentication failed: " + portal.getLoadError().getMessage()).show();
```

Comment:
You can get rid of `showMessage` if it's not needed.

Reply:
👍  I've removed the rest of the method calls and replaced with the same style 'new Alert().show'